indico/modules/receipts/controllers/event.py

```diff
@@ -82,7 +82,7 @@ def _process_img(preview):
             preview = preview.convert('RGB')
         preview = square(preview)
         if preview.height > IMAGE_PREVIEW_SIZE:
-            preview = preview.resize((IMAGE_PREVIEW_SIZE, IMAGE_PREVIEW_SIZE), resample=Image.BICUBIC)
+            preview = preview.resize((IMAGE_PREVIEW_SIZE, IMAGE_PREVIEW_SIZE), resample=Image.Resampling.BICUBIC)
```

Comment:
Is the old name deprecated? I didn't find anything, except this change from 10.0 which sounds like both names are fine to use:

|   Removed   |   Use instead   |
| --- | --- |
|   `Image.LINEAR`   |   `Image.BILINEAR` or `Image.Resampling.BILINEAR`   |
|   `Image.CUBIC`   |   `Image.BICUBIC` or `Image.Resampling.BICUBIC`   |
|   `Image.ANTIALIAS`   |   `Image.LANCZOS` or `Image.Resampling.LANCZOS`   |

Reply:
I was getting this error message: `Module 'PIL.Image' has no 'BICUBIC' member` and when I inspect the PIL code they have only enum Resampling.BICUBIC and not as a member. Did you not see this error?

Reply:
```
In [2]: from PIL import Image

In [3]: Image.BICUBIC
Out[3]: 3
```

I think it's produced by this awful piece of code in Image.py:

```py
    parent_name = __name__.rpartition(".")[0]
    for plugin in _plugins:
        try:
            logger.debug("Importing %s", plugin)
            __import__(f"{parent_name}.{plugin}", globals(), locals(), [])
        except ImportError as e:
            logger.debug("Image: failed to import %s: %s", plugin, e)
```

Reply:
Do you prefer to leave it as Image.BICUBIC?

Reply:
But I'm fine w/ using he more verbose option as long as it's updated everywhere (just 2 places in the codebase AFAICT)

Reply:
Yeah, I updated it in the other places as well